cdap-app-fabric/src/test/java/co/cask/cdap/metadata/MetadataHttpHandlerTest.java

```diff
@@ -662,8 +665,40 @@ public boolean apply(Map.Entry<String, String> input) {
                       DatasetSystemMetadataWriter.BATCH_TAG,
                       DatasetSystemMetadataWriter.EXPLORE_TAG),
       dsSystemTags);
+
     Map<String, String> dsSystemProperties = getProperties(datasetInstance, MetadataScope.SYSTEM);
-    Assert.assertEquals(KeyValueTable.class.getName(), dsSystemProperties.get("type"));
+    // Verify create time exists, and is within the past hour
+    Assert.assertTrue("Dataset create time does not exist", dsSystemProperties.containsKey("createtime"));
```

Comment:
We already have properties like `pluginversion`, I was just trying to follow the convention.